eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -219,15 +160,44 @@ jobs:
         - pwsh: npm install -g moxygen
           displayName: Install Moxygen to generate markdown for docs.microsoft.com
 
+        - pwsh: |
+            $latestVersion = "20240927.5"
```

Comment:
It's a better practice to put this under variable in line 88 as job variable and refer it.